web/openapi/openapi.yaml

```diff
@@ -270,6 +311,11 @@ paths:
                         - id: 0
                           definition: export asdf
                           instrumented: false
+                      diagnostics: []
```

Comment:
The diagnostics array should contain what is currently emitted for this pipeline definition.

Reply:
Should probably just be deleted (see my other comment).

Reply:
deleted in https://github.com/tenzir/tenzir/pull/3376/commits/324fa9d2399798c712324aa053732e7ea3a03f3e